src/nvim/insexpand.c

```diff
@@ -1722,6 +1722,12 @@ bool ins_compl_active(void)
   return compl_started;
 }
 
+/// Return True when wp is the actual completion window
+bool ins_compl_win_active(win_T *wp)
+{
+  return ins_compl_active() && !(wp->w_p_pvw || (wp->w_floating && wp->w_float_is_info));
```

Comment:
Is the `w_floating` check needed? It's not checked in the condition in `pum_set_selected`.